recipe/meta.yaml

```diff
@@ -183,18 +163,15 @@ about:
   license_family: APACHE
   summary: Common protobufs used in Google APIs
   description: |
-    This package does not include the grpc extra requirements.
     This repository contains the original interface definitions of public
     Google APIs that support both REST and gRPC protocols. Reading the original
     interface definitions can provide a better understanding of Google APIs and
     help you to utilize them more efficiently. You can also use these definitions
     with open source tools to generate client libraries, documentation,
-    and other artifacts.
-    **Deprecated Python Versions**
-    Python == 2.7
```

Comment:
I see no reason to keep this deprecation warning here, in particular since we're not building this for python 2 anyway, and the warning is over 2 years old.